network/interface.go

```diff
@@ -124,4 +124,7 @@ type Network interface {
 	NumConnectedPeers() int
 	ReachabilityStatus() string
 	HostAddrs() []string
+	Name() string
+	DHTSize() int32
```

Comment:
What is the reason for user to know the DHT size?

Reply:
How about other information?
In this level, user should look at the log, if it is important we can add a log for DHT size.

Reply:
yes, we can add it to logs or metrics also. 
about other information, these are was all of infos we currently have.
the purpose of this PR is definig peers info and network info sparated and better. about the information itself, we need to work on it in another PR, because there is more data that we can extract from network.

Reply:
Some information is useful inside the module, but not helpful to be exported. They are designed for the internal usage, like DHT size. 

We shouldn't add new API unless there's a really good reason for it.

Reply:
I agree, we can remove the DHT size.